docs/cmd/jx-gitops_annotate.md

```diff
@@ -15,9 +15,11 @@ Annotates all kubernetes resources in the given directory tree
 ### Examples
 
   # updates recursively annotates all resources in the current directory
-  jx-gitops annotate myannotate=cheese another=thing
+  jx-gitops annotate myannotation=cheese another=thing
   # updates recursively all resources
   jx-gitops annotate --dir myresource-dir foo=bar
+  # remove annotations
+  jx-gitops annotate myannotate- another-
```

Comment:
is this a standard way to annotate?
Seems quite strange

Reply:
Yes, it's how you remove an annotation with `kubectl annotate`